### DISCUSSION TITLE
Are all podman CLI options covered in the Python library

### DISCUSSION THREAD

**Discussion by eskultety** [containers/podman-py#124] [Q&A] [answered]
Hi,
we rely on the following podman CLI options in our CI scripts:
```
--env
--ulimit
--cap-add
--gidmap
--uidmap
```
We're thinking of getting rid of our Makefile and adopting your Python REST wrapper (since all the other scripts are in Python), but I could not find how the options ^above are represented in _podman-py_, can you point me the right direction please?

**Comment by jwhonce** [comment, marked as answer]
For container create() these options are given as python keyword arguments. The keywords used are the same as docker-py to ease moving scripts between the libraries.

For example:

env => environment
ulimit => ulimits
cap-add => cap_add
gidmap => group_add
uidmap => user

```python
alpine_image = client.images.pull("quay.io/libpod/alpine", tag="latest")
container = client.containers.create(
  alpine_image, command=["echo", "Hello, World"], env={"VAR1":"Value1"}
)
```
